scalameta/trees/shared/src/main/scala/scala/meta/internal/trees/package.scala

```diff
@@ -215,4 +215,10 @@ package object trees {
     if (rank == 0) clazz
     else arrayClass(ScalaRunTime.arrayClass(clazz), rank - 1)
   }
+
+  def onlyLastParamCanBeRepeated(params: List[Term.Param]): Boolean = {
+    params
+      .take(params.length - 1)
```

Comment:
Shouldn't we also have an iterator here in order not to allocate a new list?

Reply:
correct, changed :+1: